apps/system/js/wifi.js

```diff
@@ -13,6 +13,9 @@ var Wifi = {
   // If it's set to 0, wifi will never be turn off.
   screenOffTimeout: 0,
 
+  // If user wants to allow wifi off.
+  wifiSleepMode: false,
+
   // if Wifi is enabled but disconnected, try to scan for networks every
```

Comment:
Plese update the coomment here.
// When wifiSleepMode is true,  Wi-Fi will be automatically turned off during sleep to save battery power.